extensions/resteasy/deployment/src/main/java/io/quarkus/resteasy/deployment/ResteasyBuiltinsProcessor.java

```diff
@@ -69,4 +72,16 @@ private boolean isHtmlFileName(String fileName) {
         return fileName.endsWith(".html") || fileName.endsWith(".htm");
     }
 
+    @Record(STATIC_INIT)
+    @BuildStep(onlyIf = IsDevelopment.class)
+    void addAdditionalEndpointsExceptionMapper(List<NotFoundPageDisplayableAdditionalBuildItem> displayableAdditionalBuildItems,
```

Comment:
Build items variables are often named without a `BuildItem(s)` suffix. Maybe you could name this one `displayableEndpoints` or `notFoundPageDisplayableEndpoints`?